### DISCUSSION TITLE
How do I configure Renovate to query for forked lib version instead of a primary version?

### DISCUSSION THREAD

**Discussion by greg-slonim** [renovatebot/renovate#23564] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

36.23.2

### If you're self-hosting Renovate, select which platform you are using.

gitlab.com

### Was this something which used to work for you, and then stopped?

I never saw this working

## Context
- This is a dotnet (nuget) build system
- We have a nuget.config with [Package Source Mapping](https://learn.microsoft.com/en-us/nuget/consume-packages/package-source-mapping)
- We have forked Nerdbank's Gitversioning to implement some custom features and release our own artefact with a special patch version, ex. `<PackageVersion Include="Nerdbank.GitVersioning" Version="3.4.242-ttd"/>`

## Problem
Renovate looks up the `Nerdbank.GitVersioning` artefact and finds a newer version of the **regular** library, ex. `3.4.255` and produces an MR/PR upgrading to that version.

## Question
Is there any way to configure Renovate to only lookup version upgrades with a specific patch suffix? 
Another question: Does Renovate use `nuget.config` to figure out where and how to lookup artefacts based on Package Source Mapping?

### What you tried so far.

Nothing specific

### Relevant debug logs

<details><summary>Logs</summary>

```
INFO: PR created (repository=thetradedesk/teams/csx/TTD.Internal.Actions, branch=dependency_bot_nerdbank.gitversioning-3.4.x)
       "pr": 18,
       "prTitle": "Update dependency Nerdbank.GitVersioning to v3.4.255"
```

</details>

**Comment by rarkins** [comment, marked as answer]
If you configure versioning=docker for this dependency then it should enforce the suffix

**Reply by greg-slonim** [reply]
Thanks, Rhys. This appears to have worked on a test repo.
